lib/raxx.ex

```diff
@@ -94,15 +97,17 @@ defmodule Raxx do
       false
   """
   @spec request(Raxx.Request.method(), String.t() | URI.t()) :: Raxx.Request.t()
-  def request(method, url) when is_binary(url) do
-    url = URI.parse(url)
-
-    if url.query do
-      {:ok, query} = URI2.Query.decode(url.query)
-      request(method, %{url | query: query})
-    else
-      request(method, url)
-    end
+  def request(method, raw_url) when is_binary(raw_url) do
+    url = URI.parse(raw_url)
+
+    url =
+      if is_nil(url.query) && String.contains?(raw_url, "?") do
```

Comment:
This modification is to handle this issue https://github.com/elixir-lang/elixir/issues/7563